e2e/framework/clients/rest_client.go

```diff
@@ -23,7 +25,8 @@ func CreateRestAPIClient(platform enums.RookPlatformType) *RestAPIClient {
 		//Start rook_api_external server via nodePort if not it not already running.
 		_, err := rkh.GetService("rook-api-external")
 		if err != nil {
-			rkh.ResourceOperation("create", "../../data/smoke/rook_api_external.yaml")
+			path := filepath.Join(os.Getenv("GOPATH"), "src/github.com/rook/rook/e2e/data/smoke/rook_api_external.yaml")
```

Comment:
What if we avoid the relative/absolute path completely by embedding the yaml files as a string in the Go test code? We don't need the yaml files separately from the code, right?